example/screens/MapTypesExample.tsx

```diff
@@ -14,11 +14,7 @@ export default function MapTypesExample() {
   const [open, setOpen] = useState<boolean>(false);
   return (
     <View style={styles.container}>
-      <Maps.ExpoMap
-        style={{ flex: 1, width: '100%' }}
-        provider={provider}
-        mapType={mapType}
-      />
+      <Maps.ExpoMap style={{ flex: 1, width: '100%' }} provider={provider} mapType={mapType} />
```

Comment:
Again it looks like something is very inconsistent between our prettier setting, we should try to fix this issue 🤔